runtime/lua/vim/diagnostic.lua

```diff
@@ -1276,12 +1276,10 @@ M.handlers.signs = {
         local sign = vim.fn.sign_getdefined(name)[1]
         if sign then
           local severity = M.severity[v:upper()]
-          vim.deprecate(
+          vim._deprecate(
             'Defining diagnostic signs with :sign-define or sign_define()',
-            'vim.diagnostic.config()',
             '0.12',
-            nil,
-            false -- suppress backtrace
+            { alternative = 'vim.diagnostic.config()' }
```

Comment:
Note: For now `vim._deprecate` does not have `backtrace` in opts, so this will no longer suppress backtrace.

Though it'd be even better if we can allow suppressing backtraces (not mandatory), adding stacktrace back here for this particular instance would be also fine, as the stacktrace information (which we think is useless) will be shown only in `:checkhealth` rather than as a message.

Reply:
Do you anticipate suppressing backtracea in checkhealth being a real usecase for us?